# This message is for storing bounding boxes in a generic format
float64 x
float64 y
float64 z
float64 height_radius
float64 width_radius
int16 class_id